### ISSUE TITLE
please update to 1.17

### ISSUE THREAD

**Issue by Nicolas-GE** [rebane2001/cr3stal#3]

**Comment by rebane2001** [comment]
I guess I'll have to update it to 1.17

**Comment by ayunami2000** [comment]
I guess you have to update it to 1.17

**Comment by leon0399** [comment]
I guess you have to update it to 1.17.1

**Comment by Nicolas-GE** [comment]
i tried to update it to 1.17.1 but i can't, so if you saw me fork it twice then delete it that's why :P

(i don't really know what i'm doing)

**Comment by rebane2001** [comment]
Pretty much all you need to do is da63380b572d4d00edb479e6f665b9e5757539f7

**Comment by Nicolas-GE** [comment]
yeah i tried that but with 1.17.1 being java 16 and all probabely gives errors

(37 errors to be exact, and i tried setting everything ftom java 8 to java 16, didn't work)

**Comment by Lord-Memester** [comment]
Hopefully, this is still WIP. Otherwise, I might just have to learn java. ;p

**Comment by Nicolas-GE** [comment]
might as well port to 1.18.1 at that point 

edit: doing a 1.17.x version on the way there would be nice but i don't think much pepole play on 1.17 anymore
